controllers/AuthController.js

```diff
@@ -0,0 +1,58 @@
+import bcrypt from 'bcrypt';
+import db from '../models';
+import Authorization from '../middlewares/Authorization';
+
+const { User } = db;
+/**
+ * @exports AuthController
+ * @class AuthController
+ * @description Handles the user registartion and Sigin
+ * */
+class AuthController {
+  /**
+     * Authenticate and Login the User to the application
+     * @async
+     * @param  {object} req - Request object
+     * @param {object} res - Response object
+     * @param {function} next - next middleware
+     * @return {json} Returns json object
+     * @static
+     */
+  static async login(req, res, next) {
+    const { email, password } = req.body;
+    try {
+      User.findOne({ where: { email } }).then(async (foundUser) => {
```

Comment:
Mixing `then` syntax with `async/await` does not seem optimal here. It misses the point of the flatter and more readable code structure that `async/await` offers. You could have:
```
const foundUser = await User.findOne({ where: { email } });
if (!foundUser) {
...
}
...
```